job-service-db/src/main/resources/procedures/reportComplete.sql

```diff
@@ -53,6 +53,9 @@ BEGIN
     -- Get the job id
     v_job_id = internal_get_job_id(in_task_id);
 
+    -- Adding sub-task details into a specific report table for analysis purpose ( SCMOD-10455 )
+    INSERT INTO subtask_report (partition_id, task_id, status, create_date) values (in_partition_id, in_task_id, 'Completed', now());
```

Comment:
- Include the job id in the table as well.
- Rename `create_date` to `report_date`.
- Instead of `now()` use `now() AT TIME ZONE 'UTC'`.